articles/azure-monitor/alerts/action-groups.md

```diff
@@ -25,6 +25,10 @@ Global requests from clients can be processed by action group services in any re
 - Multiple alert rules can use the same action group.
 - Action Groups are defined by the unique set of actions and the users to be notified. For example, if you want to notify User1, User2 and User3 by email for two different alert rules, you only need to create one action group which you can apply to both alert rules.
 
+> [!NOTE]  
+> Alert and notification are separate concepts. Only having an alert may suffice your needs. You need to configure notification only if you need to send a voice call, SMS, or email
+> 
+
```

Comment:
I would not add this note here. I would add a clarification in the first paragraph of the article, and change it to say:

When Azure Monitor data indicates that there might be a problem with your infrastructure or application, an alert is triggered. You can use an action group to send a notification such as a voice call, SMS or email when the alert is triggered in addition to the alert itself. Action groups are a collection of notification preferences and actions. Azure Monitor, Azure Service Health, and Azure Advisor use action groups to notify users about the alert and take an action.